Dockerfile

```diff
@@ -24,8 +24,13 @@ COPY ankerctl.py /app/
 COPY static /app/static/
 COPY libflagship /app/libflagship/
 COPY cli /app/cli/
+COPY specification /app/specification
+COPY web /app/web
+COPY transwarp /app/transwarp
+COPY examples/ankermake-mqtt.crt /app/ankermake-mqtt.crt
```

Comment:
Thanks for this PR. 

Whats the thinking behind adding these extra folders? As far as I can tell they aren't required for running the container. 🤔 

specification/
web/
transwarp/
examples/